src/feature.js

```diff
@@ -356,6 +360,67 @@ var feature = function (arg) {
 
   ////////////////////////////////////////////////////////////////////////////
   /**
+   * Set style(s) from array(s).  For each style, the array should have one
+   * value per data item.  The values are not converted or validated.  Color
+   * values should be objects with r, g, b values on a scale of [0, 1].  If
+   * invalidate values are given the behavior is undefined.
+   *   For features where this._subcomponentStyles is an object and a style
```

Comment:
Since this is a public API, listing private API (from documentation point of view) could be confusing to the user.